sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.processing.service.ts

```diff
@@ -115,7 +115,10 @@ export class SFASIntegrationProcessingService {
               const errorDescription = `Error processing record line number ${record.lineNumber}`;
               this.logger.error(errorDescription);
               this.logger.error(error);
-              result.summary.push(errorDescription);
+              this.logger.error(error.detail);
+              result.summary.push(
+                `${errorDescription}\nerror: ${error.message}\ndetail: ${error.detail}`,
+              );
```

Comment:
We do not need the runtime error trace to be part of result summary message and also we don't have to manually add a line break. I would say the existing code as it is now is sufficient.

Reply:
The second part of the AC is to "check if there is some log improvement to make the messages clear." The errors in the story shows "Error processing record line number 44", but it doesn't show what's the cause of the error. Line 120 is to address this AC and make the cause of error explicit. I think it's necessary to log `error.message` and `error.detail` to provide more information to the error message than ""Error processing record line number 44". Please let me know if we really want to keep error in the original way and I will revert it back. Thanks.

Reply:
As I mentioned in earlier comment https://github.com/bcgov/SIMS/pull/3633#discussion_r1720286998 we can provide a momentary solution this way.